pynest/nest/lib/hl_api_info.py

```diff
@@ -356,6 +357,76 @@ def GetStatus(nodes, keys=None, output=''):
     See Also
     --------
     SetStatus
+
+    Examples
+    --------
+    *For nodes:*
+
+    >>>    nest.GetStatus(nodes)
+           ({'archiver_length': 0,
+             'beta_Ca': 0.001,
+             ...
+             'global_id': 1,
+             ...
+             'vp': 0},
+            ...
+            {'archiver_length': 0,
+             'beta_Ca': 0.001,
+             ...
+             'global_id': 3,
+             ...
+             'vp': 0})
+
+    >>>    nest.GetStatus(nodes, 'V_m')
+           (-70.0, -70.0, -70.0)
+
+    >>>    nest.GetStatus(nodes, ['V_m', 'C_m'])
+           ((-70.0, 250.0), (-70.0, 250.0), (-70.0, 250.0))
+
+    >>>    nest.GetStatus(nodes, ['V_m', 'C_m'], output='json')
+           '[[-70.0, 250.0], [-70.0, 250.0], [-70.0, 250.0]]'
+
+    *For connections:*
+
+    >>>    nest.GetStatus(conns)
+           ({'delay': 1.0,
+             'port': 0,
+             'receptor': 0,
+             'sizeof': 32,
+             'source': 1,
+             'synapse_id': 15,
+             'synapse_model': 'static_synapse',
+             'target': 1,
+             'target_thread': 0,
+             'weight': 1.0},
+            ...
+            {'delay': 1.0,
+             'port': 8,
+             'receptor': 0,
+             'sizeof': 32,
+             'source': 3,
+             'synapse_id': 15,
+             'synapse_model': 'static_synapse',
+             'target': 3,
+             'target_thread': 0,
+             'weight': 1.0})
+
+    >>>    nest.GetStatus(conns, 'weight')
+           (1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0)
+
+    >>>    nest.GetStatus(conns, ['source', 'delay'])
+           ((1, 1.0),
+            (1, 1.0),
+            (1, 1.0),
+            (2, 1.0),
+            (2, 1.0),
+            (2, 1.0),
+            (3, 1.0),
+            (3, 1.0),
+            (3, 1.0))
+
+    >>>    nest.GetStatus(conns, ['source', 'delay'], output='json')
+           '[[1, 1.0], [1, 1.0], [1, 1.0], [2, 1.0], [2, 1.0], [2, 1.0], [3, 1.0], [3, 1.0], [3, 1.0]]'
```

Comment:
Could we break this line at 80 characters?

Reply:
Done!

Reply:
Thanks!